.clang-format

```diff
@@ -0,0 +1,7 @@
+#Official Tool: clang-format version 8.0.0
+BasedOnStyle: google
+SortIncludes: false
+AlignConsecutiveAssignments: true
+AllowShortCaseLabelsOnASingleLine: true
+AllowShortIfStatementsOnASingleLine: true
+#AllowShortLambdasOnASingleLine: true
```

Comment:
Why is this commented? If we decide the official tool is clang-format 8.0.0, we can enable this, can't we?

Reply:
Yeah we can remove that line again I think it is clang-9

Reply:
It is an option introduced in Clang 9.  @crtrott please remove the line or add a comment that you will uncomment when we move to Clang-Format version 9.

Reply:
I deleted the line